crates/cheatnet/src/forking/state.rs

```diff
@@ -150,11 +165,13 @@ impl StateReader for ForkStateReader {
         if let Some(cache_hit) = self.cache.borrow().get_nonce_at(&contract_address) {
             return Ok(cache_hit);
         }
-
-        match sync(
+        debug!("Getting nonce for contract_address = {contract_address:?}");
+        let result = self.runtime.block_on(
             self.client
                 .get_nonce(self.block_id(), Felt::from_(contract_address)),
-        ) {
+        );
+        debug!("Result getting nonce for contract_address = {contract_address:?} = {result:?}");
```

Comment:
So it will look e.g. like: `Result getting nonce for contract_address = 0x123 = 0x1`.
Could we improve readability of it 😅?
Applies to other result logs, because the pattern seems the same.